plugins/API/ProcessedReport.php

```diff
@@ -363,9 +363,13 @@ public function getProcessedReport($idSite, $period, $date, $apiModule, $apiActi
 
         list($newReport, $columns, $rowsMetadata, $totals) = $this->handleTableReport($idSite, $dataTable, $reportMetadata, $showRawMetrics, $formatMetrics);
 
-        foreach ($columns as &$name) {
-            $name = ucfirst($name);
-        }
+	if (function_exists('mb_substr)) {
+            foreach ($columns as &$name) {
+                if (substr($name, 0, 1) === mb_substr($name, 0, 1) {
```

Comment:
See http://php.net/manual/de/mbstring.overload.php

Using [a-zA-Z] wouldn't be correct either, as some languages using umlauts should be shown ucfirst.
Not sure how to solve that best, but maybe it would be better to have that "kind" of configurable per language?
We could add that to the Intl plugin and have a translation key there providing the information or something like that.